src/experiment.js

```diff
@@ -3,31 +3,49 @@ const path = require('path');
 const { FILEDIR } = require('./dirs');
 const { authMiddleware } = require('./authentication');
 
+const { timeStamp } = require('./serverUtils');
+
+var Config = require('./models/Config.model');
+
+const collection = 'experiment_data'
+
 module.exports = (router = new Router()) => {
-// Get config file
+  //get all config documents from mongo.
+  router.get('/', authMiddleware);
```

Comment:
Can we add a route that gets a specific config by id?
Like: `/config/4` gets config 4?